### PR TITLE
Support specifying API endpoins via java props

### PR DESCRIPTION
**Pull request by deathbeam**
- Add support fo specifying runelite.session.url java prop to be used
instead of RuneLite session base
- Add support fo specifying runelite.http-service.url java prop to be used
instead of RuneLite api base url
- Add support fo specifying runelite.ws.url java prop to be used
instead of RuneLite websocket url
- Add support for specifying runelite.static.url java prop to be used
instead of RuneLite s.r.n url

Signed-off-by: Tomas Slusny <slusnucky@gmail.com>

### CODE REVIEW

File: http-api/src/main/java/net/runelite/http/api/RuneLiteAPI.java [hunk 1]
```diff
@@ -102,6 +102,13 @@ public static HttpUrl getApiRoot()
 
 	public static HttpUrl getApiBase()
 	{
+		final String prop = System.getProperty("runelite.api.url");
+
+		if (prop != null && !prop.isEmpty())
```
Comment (chivesrs) [inline http-api/src/main/java/net/runelite/http/api/RuneLiteAPI.java]: https://google.github.io/guava/releases/19.0/api/docs/com/google/common/base/Strings.html#isNullOrEmpty(java.lang.String)

Comment (deathbeam) [reply http-api/src/main/java/net/runelite/http/api/RuneLiteAPI.java]: I am aware of that, we don't use Guava in http-api

**Comment by Adam-** [comment]
nothing for getApiRoot()? Or the srn base?

**Comment by deathbeam** [comment]
srn base is not important at all, and API root is just for session counter, and also is not very flexible (because its not possible to change the session suffix) and I do not wanted to refactor that.

**Comment by deathbeam** [comment]
Updated with configurable s.r.n url

**Pull request merged**